runtime/src/iree/builtins/ukernel/arch/arm_64/mmt4d_arm_64_i8mm.c

```diff
@@ -175,7 +175,8 @@ iree_uk_mmt4d_tile_s8s4s32_2x8x16_to_8x8x16_arm_64_i8mm(
     void* IREE_UK_RESTRICT out_tile, const void* IREE_UK_RESTRICT lhs_panel,
     const void* IREE_UK_RESTRICT rhs_panel,
     const iree_uk_mmt4d_params_t* params, int M0) {
-  IREE_UK_ASSERT(M0 >= 2 && M0 <= 8 && iree_uk_is_po2_u32(M0));
+  // We support M0 up to 4 in order to fit within the register budget.
```

Comment:
Also update this function's name: `..._to_4x8x16_...`

Reply:
Oh yes, nice catch.